packages/buckram/assets/styles/components/specials/_contributors.scss

```diff
@@ -6,63 +6,55 @@
 //                           Contributors                                     //
 //============================================================================//
 
-div.contributors {
+.contributors {
   border-top: 1px solid;
-  margin-top: 2rem;
+  margin-top: $contributors-margin-top;
+  font-family: $body-font-family;
 
-  &.page {
-    border-top: 0;
-
-    h3 {
-      font-family: $font-1;
-      font-size: $h1-font-size;
-    }
-
-    .contributor_name {
-      font-size: 25px;
-    }
-  }
-
-  .contributor_name_and_links {
-    margin-top: 2rem;
+ .about-authors {
+    margin-top: $contributors-margin-top;
+    margin-bottom: $contributors-margin-bottom;
+    text-align: left;
   }
 
-  .about-authors {
-   margin: 2rem 0;
-   text-align: left;
+  // TODO: replace clearfix hack with better CSS Grid solution when Prince supports grid layout
+  .contributor__name__and__links:after {
+    content: "";
+    display: table;
+    clear: both;
   }
 
-  .contributor_profile_picture {
-    float: left;
+  .contributor__profile__picture {
+    float: $contributor-profile-picture-float;
+    margin: $contributor-profile-picture-margin;
+    max-height: 200px;
```

Comment:
Not sure the best way to handle this. All profile images are 400x400px. We want to display them in the webbook with as 200x200px images (with retina support). One potential problem is that there is already CSS for #content img elements with the following rules: max-width: 100%, height: auto. Open to suggestions/ideas here ...

Reply:
Ideally, I'd like to set max-width values for webbook and exports separately. Potentially even using percentage width ...